lib/knapsack_pro/utils.rb

```diff
@@ -5,5 +5,13 @@ class Utils
     def self.unsymbolize(obj)
       JSON.parse(obj.to_json)
     end
+
+    def self.now
```

Comment:
Just a personal opinion more than anything else - I'm not convinced `Utils.now` makes it clear that it returns the current time, maybe `time_now`? But feel free to ignore.